EventFilter/Utilities/plugins/TcdsRawToDigi.cc

```diff
@@ -71,9 +71,16 @@ void TcdsRawToDigi::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
 
   TCDSRecord tcdsRecord;
   if (rawdata.isValid()) {
-    const FEDRawData& tcdsData = rawdata->FEDData(FEDNumbering::MINTCDSuTCAFEDID);
-    if (tcdsData.size() > 0) {
-      tcdsRecord = TCDSRecord(tcdsData.data());
+    uint16_t selectedId = 0;
+    for (uint16_t fedId = FEDNumbering::MINTCDSuTCAFEDID; fedId <= FEDNumbering::MINTCDSuTCAFEDID; fedId++) {
```

Comment:
The start and end conditions of this loop seem to be the same - is it actually entered?

Reply:
Yes, it's a mistake, it should be MAX...